core/src/main/kotlin/ch/ergon/dope/resolvable/clause/model/SetClause.kt

```diff
@@ -0,0 +1,41 @@
+package ch.ergon.dope.resolvable.clause.model
+
+import ch.ergon.dope.DopeQuery
+import ch.ergon.dope.resolvable.clause.IUpdateSetClause
+import ch.ergon.dope.resolvable.clause.IUpdateUseKeysClause
+import ch.ergon.dope.resolvable.expression.TypeExpression
+import ch.ergon.dope.resolvable.expression.unaliased.type.Field
+import ch.ergon.dope.resolvable.formatToQueryString
+import ch.ergon.dope.validtype.ValidType
+
+class SetClause(
+    private val fieldAssignment: Pair<Field<out ValidType>, TypeExpression<out ValidType>>,
```

Comment:
Please ensure that the field and the value have the same type. (We should not allow assigning an integer to a strgin-field)